app/helpers/poem_helper.rb

```diff
@@ -1,8 +1,20 @@
 module PoemHelper
+  # ToDo 他の箇所でも使うようになったら(コメントとか)
+  # ここじゃない場所に移した方が良さそう
   def markdown(text)
-    renderer = Redcarpet::Render::HTML.new(filter_html: true)
+    renderer = Redcarpet::Render::HTML.new
     extensions = {}
     markdown = Redcarpet::Markdown.new(renderer, extensions)
-    markdown.render(text).html_safe
+    markdown.render(emojify(text)).html_safe
+  end
+
+  def emojify(content)
+    h(content).to_str.gsub(/:([\w+-]+):/) do |match|
+      if emoji = Emoji.find_by_alias($1)
+        %(<img alt="#$1" src="#{image_path("emoji/#{emoji.image_filename}")}" style="vertical-align:middle" width="20" height="20" />)
```

Comment:
__[rubocop]__
* Avoid the use of Perl-style backrefs. [:ref](https://github.com/bbatsov/ruby-style-guide#no-perl-regexp-last-matchers)
* Replace interpolated variable `$1` with expression `#{$1}`. [:ref](https://github.com/bbatsov/ruby-style-guide#curlies-interpolate)
* Line is too long. [134/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)